### PR TITLE
Various doc updates

### PR DESCRIPTION
**Pull request by jits**
See individual commits for details.

### CODE REVIEW

File: docs/installation/installing-with-npm.md [hunk 1]
```diff
@@ -34,7 +38,7 @@ You will need to import a couple of things into your project before you can star
 
 ## Importing styles
 
-To build the stylesheet you will need a pipeline set up to compile [Sass](https://sass-lang.com/) files to CSS. We recommend using [gulp](https://gulpjs.com/) and [gulp-sass](https://www.npmjs.com/package/gulp-sass) however you can use any tools that you are familiar with.
+To build the stylesheet you will need a pipeline set up to compile [Sass](https://sass-lang.com/) files to CSS. We recommend using the [sass](https://www.npmjs.com/package/sass) npm package, however you can use any tools that you are familiar with.
 
 You need to import the Our Future Health design system toolkit styles into the main Sass file in your project. You should place the below code before your own Sass rules (or Sass imports).
```
Comment (angusbayley) [inline docs/installation/installing-with-npm.md:44]: FYI issue with the following line

`@import 'node_modules/ofh-design-system-toolkit/packages/ofh';`

should be `@import 'PATH/TO/node_modules/ofh-design-system-toolkit/packages/ofh';`

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Our Future Health design system toolkit contains the code you need to start buil
 
 ## Guidance
 
-Visit the [NHS digital service manual](https://service-manual.nhs.uk/) for examples of components and guidance for when to use them.
+Visit the [docs site](https://designsystem.ourfuturehealth.org.uk/) for examples of components and guidance for when to use them.
```
Comment (angusbayley) [inline README.md:12]: "design system site" a bit clearer here maybe?

Comment (angusbayley) [reply README.md:12]: or "Design system docs website"

**Review by angusbayley — COMMENTED** [review]
A couple of comments, but 👍

File: docs/installation/installing-with-npm.md (hunk 1 above)
Comment (jits) [reply docs/installation/installing-with-npm.md:44]: 👍🏽 Will update.

File: README.md (hunk 2 above)
Comment (jits) [reply README.md:12]: Yes, good shout – I'll make this consistent with the terminology later on in the README (i.e. `Design system docs website`, as you suggest).

**Pull request merged**